source/includes/read/change-streams.kt

```diff
@@ -0,0 +1,57 @@
+import com.mongodb.ConnectionString
+import com.mongodb.MongoClientSettings
+import com.mongodb.client.model.Aggregates
+import com.mongodb.client.model.Filters
+import com.mongodb.client.model.Updates
+import com.mongodb.client.model.changestream.FullDocument
+import com.mongodb.kotlin.client.MongoClient
+
+// start-data-class
+data class Restaurant(
+    val name: String,
+    val cuisine: String,
+)
+// end-data-class
+
+fun main() {
+    val uri = "<connection string URI>"
+
+    val settings = MongoClientSettings.builder()
+        .applyConnectionString(ConnectionString(uri))
+        .retryWrites(true)
```

Comment:
Why include this?  It's already on by default.

Reply:
To your top-level comment (questions about how to review this) we usually request technical reviews to confirm that the technical language in the document is accurate, as well as whether the code samples are up to standard.

As for the `retryWrite` line, I can remove that.

Reply:
ok, let me know when you have removed it.

Reply:
I've removed it!